server/controller.js

```diff
@@ -0,0 +1,23 @@
+const path = require("path");
+const pathToDatabase = path.join(__dirname, '../database/indexDB.js')
+const db = require(pathToDatabase);
+console.log(db);
+
+
+const getSimilarListing = (req, res) => {
+  db.getSimilarListing((error, results, fields) => {
+    if (error) console.log(error);
+    res.send(results);
+  })
+};
+
+const getThingsToDo = (req, res) => {
+  db.getThingsToDo((error, results, fields) => {
+    if (error) console.log(error);
+    res.send(results);
+  })
+};
+
+module.exports.db=db;
```

Comment:
Might be cleaner to put db, getSimilarListing, and getThingsToDo all in one object, like:
```
module.exports = {
  db,
  getSimilarListing,
  getThingsToDo,
};
```

Reply:
Thanks for your comments and helping me to tie up loose ends. Please review :+1